.gitignore

```diff
@@ -0,0 +1,2 @@
+*.pyc
+.DS_Store
```

Comment:
@spwilson2 Agh, yeah, sorry. Brainfart there. This really does need to sort of "match" what's in the book though, at least until O'Reilly lets me get a second edition out (hopefully?). Ideally, the original import statement should be in place, and individual files should be able to run independently. I added another potential solution, below. 

We could also do something like

```python
#Some comment here
from bs4 import BeautifulSoup as BeautifulSoup_original

#Some comment here like "BS4 4.4.0 came out a month after the book published! Sorry guys!"
def BeautifulSoup(your_markup, parser='html.parser'):
    BeautifulSoup_original(your_markup, parser)
```